fluentd/configs.d/openshift/filter-k8s-meta.conf

```diff
@@ -7,9 +7,18 @@
   ssl_partial_chain "#{ENV['SSL_PARTIAL_CHAIN'] || 'true'}"
 </filter>
 
-<filter kubernetes.**>
+<filter kubernetes.journal.**>
+  @type parse_json_field
+  merge_json_log "#{ENV['MERGE_JSON_LOG'] || 'false'}"
+  preserve_json_log "#{ENV['PRESERVE_JSON_LOG'] || 'true'}"
+  # use MESSAGE as the field to parse for json unless USE_JOURNAL is explicitly set to false
+  json_fields "#{ENV['JSON_FIELDS'] || (ENV['USE_JOURNAL'] == 'false' ? 'log,MESSAGE' : 'MESSAGE,log')}"
+</filter>
+
+<filter kubernetes.var.log.containers.**>
   @type parse_json_field
   merge_json_log "#{ENV['MERGE_JSON_LOG'] || 'false'}"
   preserve_json_log "#{ENV['PRESERVE_JSON_LOG'] || 'true'}"
-  json_fields "#{ENV['JSON_FIELDS'] || (ENV['USE_JOURNAL'] == 'true' ? 'MESSAGE,log' : 'log,MESSAGE')}"
+  # use log as the field to parse for json unless USE_JOURNAL is explicitly set to true
+  json_fields "#{ENV['JSON_FIELDS'] || (ENV['USE_JOURNAL'] == 'true' ? 'MESSAGE,log' : 'log,MESSAGE' )}"
```

Comment:
is the use journal check still necessary here? do we expect to have a `/var/log/containers/*` entry that would use anything other than `'log,MESSAGE'`?

Reply:
It doesn't make much sense.  I'll get rid of it.